server/api/api/endpoints/nuclio.py

```diff
@@ -587,8 +587,16 @@ def _handle_nuclio_deploy_status(
     if state in ["error", "unhealthy"]:
         logger.error(f"Nuclio deploy error, {text}", name=name)
 
-    internal_invocation_urls = status.get("internalInvocationUrls", [])
-    external_invocation_urls = status.get("externalInvocationUrls", [])
+    internal_invocation_urls = (
+        status.get("internalInvocationUrls")
+        if status.get("internalInvocationUrls")
+        else []
+    )
+    external_invocation_urls = (
+        status.get("externalInvocationUrls")
+        if status.get("externalInvocationUrls")
+        else []
+    )
```

Comment:
Why is this different than `status.get("externalInvocationUrls", [])`

Reply:
@TomerShor because status["externalInvocationUrls"] can be equal to empty string